docs/configuration.asciidoc

```diff
@@ -348,9 +352,22 @@ otherwise defaults to `nil`.
 | `ELASTIC_APM_FRAMEWORK_VERSION` | `framework_version` | Depending on framework
 |============
 
-Version number of the used framework.
-For Ruby on Rails and Sinatra, this defaults to the used version of the framework,
-otherwise, the default is `nil`.
+[float]
```

Comment:
I think you deleted the body of the option above by accident?

Reply:
😬